pallets/dapp-staking-v3/src/lib.rs

```diff
@@ -520,8 +520,8 @@ pub mod pallet {
                 reward_portion: vec![Permill::from_percent(100 / num_tiers); num_tiers as usize],
                 slot_distribution: vec![Permill::from_percent(100 / num_tiers); num_tiers as usize],
                 tier_thresholds: (0..num_tiers)
-                    .map(|i| TierThreshold::FixedTvlAmount {
-                        amount: (10 * i).into(),
+                    .map(|i| TierThreshold::FixedPercentage {
+                        required_percentage: Perbill::from_percent(i),
```

Comment:
Small improvement suggestion - using this logic (and legacy one), the first threshold would be 0%. Also, it kind of makes sense to revert the iterator since we want higher thresholds first?

Not an issue introduced here, it seems like one I did before 🙂